concourse/steps/component_descriptor.mako

```diff
@@ -79,6 +79,7 @@ component_v2.resources.append(
       type=cm.AccessType.OCI_REGISTRY,
       imageReference='${image_descriptor.image_reference()}' + ':' + effective_version,
     ),
+    labels=${image_descriptor.component_descriptor_labels()}
```

Comment:
does this not require a fallback value (e.g. `None`, or `()`)?

Reply:
That default value is provided by our AttributeSpec implementation.

Reply:
good point. however: the tuple literal (`()`) is not valid yaml, is it? :-/
I know, I suggested to use tuple instead of list (however, I _asked_ whether there would be a problem if switching to it^^)